blockchain/My-Wallet-V3-iOS
# fix(Strings): Login URL | Change all .info login copy to .com

## Description ##

Updates all instances of `blockchain.info/login` to now use `login.blockchain.com`. This PR also includes updating `build-js.sh` to use `git checkout` rather than the `git co` alias. 

## How to Test ## 

1. Build and run
2. Tap `Login`.
3. Observe the corrected string value.
4. Create a wallet and/or login
5. Tap `Login to Web Wallet`
6. Observe the corrected string value.

For the `build-js.sh` script, please run the script and observe that it completes successfully. 

## Screenshots ##

<img width="342" alt="screen shot 2018-07-25 at 7 43 19 am" src="https://user-images.githubusercontent.com/41585563/43211611-cd4a3cdc-8fe6-11e8-8a48-0f55c6a31a42.png">
<img width="344" alt="screen shot 2018-07-25 at 8 32 50 am" src="https://user-images.githubusercontent.com/41585563/43211618-cfa31788-8fe6-11e8-9dfe-6978384e1a5b.png">
